lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -148,6 +148,43 @@ e2function table entity:keyvalues()
 	return ret
 end
 
+e2function number entity:setEditProperty(string key, string value)
+	if not IsValid(this) then return self:throw("Invalid entity!", 0) end
+	if not isOwner(self, this) then return self:throw("You do not own this entity!", 0) end
+	if not this.Editable then return self:throw("Tried to edit non-editable entity!", 0) end
+
+	return this:SetNetworkKeyValue(key, value) and 1 or 0
+end
+
+e2function string entity:getEditProperty(string key)
+	if not IsValid(this) then return self:throw("Invalid entity!", "") end
+	if not this.Editable then return self:throw("Tried to access non-editable entity!", "") end
+
+	return tostring(this:GetNetworkKeyValue(key) or "")
+end
+
+__e2setcost(30)
+
+e2function table entity:getEditData()
+	if not IsValid(this) then return self:throw("Invalid entity!", {}) end
+	if not this.Editable then return self:throw("Tried to access non-editable entity!", {}) end
+
+	local ret = E2Lib.newE2Table()
+	for k, _ in pairs(this:GetEditingData()) do
+		ret.s[k] = 1
+		ret.stypes[k] = "n"
```

Comment:
This unfinished?

Reply:
Wasn't sure what to do with the table. Wanted it as a LUT as then users could simply have something like
```py
if(Entity:getEditData()[DesiredKey, number]) {
    Entity:setEditProperty(DesiredKey, Value)
}
```
The actual output itself isn't very useful besides having what the type is.
![image](https://github.com/wiremod/wire/assets/20892685/c6ae6378-5da4-4c24-9230-22cd76c15565)